docs/contributing/crd-source.md

```diff
@@ -88,6 +88,7 @@ Apply this to register the CRD
 $ kubectl apply --validate=false -f docs/contributing/crd-source/crd-manifest.yaml
 customresourcedefinition.apiextensions.k8s.io "dnsendpoints.externaldns.k8s.io" created
 ```
+Or alternatively deploy the external-dns helm chart with value `deployCrd=true`.
```

Comment:
This should reflect Helm best practices of using `--skip-crds` in production and manually managing the CRDs, wioth the chart CRDs being used for testing and as a lookup.